### PR TITLE
update tsdoc for exported registrations

### PR DESCRIPTION
**Pull request by chrisdholt**
# Pull Request

## 📖 Description
The previous export from `fast-components` was an element. With the latest changes in vNext, that's changed to be a registration for a design system. This updates comments to better reflect that implementation.

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/src/design-system-provider/index.ts [hunk 1]
```diff
@@ -1029,7 +1029,7 @@ class DesignSystemProvider extends FoundationElement {
  *
  * @public
  * @remarks
- * HTML Element: \<fast-design-system-provider\>
+ * Generates HTML Element: \<fast-design-system-provider\>
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components/src/design-system-provider/index.ts:1032]: Looks like this one needs a general comment update.

File: packages/web-components/fast-components/src/radio/index.ts [hunk 2]
```diff
@@ -6,13 +6,13 @@ import {
 import { radioStyles as styles } from "./radio.styles";
 
 /**
- * The FAST Radio Element. Implements {@link @microsoft/fast-foundation#Radio},
- * {@link @microsoft/fast-foundation#radioTemplate}
+ * A function that returns a {@link @microsoft/fast-foundation#Radioi} registration for configuring the component with a DesignSystem.
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components/src/radio/index.ts:9]:
```suggestion
 * A function that returns a {@link @microsoft/fast-foundation#Radio} registration for configuring the component with a DesignSystem.
```

File: packages/web-components/fast-components/src/toolbar/index.ts [hunk 3]
```diff
@@ -29,8 +29,8 @@ export class Toolbar extends FoundationToolbar {
 }
 
 /**
- * The FAST toolbar Custom Element. Implements {@link @microsoft/fast-foundation#Toolbar},
- * {@link @microsoft/fast-foundation#ToolbarTemplate}
+ * A function that returns a {@link @microsoft/fast-foundation#Toolbar} registration for configuring the component with a DesignSystem.
+ * Implements {@link @microsoft/fast-foundation#ToolbarTemplate}
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components/src/toolbar/index.ts:33]: For consistency, below it should probably say "Generates HTML Element: \<fast-toolbar\>"

**Review by EisenbergEffect — CHANGES_REQUESTED** [review]
Just a few minor things that looked like they were missed in the mass update.

File: packages/web-components/fast-components/src/radio/index.ts (hunk 2 above)
Comment (chrisdholt) [reply packages/web-components/fast-components/src/radio/index.ts:9]: DOH

File: packages/web-components/fast-components/src/accordion/index.ts [hunk 4]
```diff
@@ -4,13 +4,13 @@ import { accordionStyles as styles } from "./accordion.styles";
...
- * The FAST Accordion Element. Implements {@link @microsoft/fast-foundation#Accordion},
- * {@link @microsoft/fast-foundation#accordionTemplate}
+ * A function that returns a {@link @microsoft/fast-foundation#Accordion} registration for configuring the component with a DesignSystem.
+ * Implements {@link @microsoft/fast-foundation#accordionTemplate}
  *
  *
  * @public
  * @remarks
- * HTML Element: \<fast-accordion\>
+ * Generatese the HTML Element: \<fast-accordion\>
```
Comment (janechu) [inline packages/web-components/fast-components/src/accordion/index.ts:13]: For spelling and consistency.

```suggestion
 * Generates HTML Element: \<fast-accordion\>
```

File: packages/web-components/fast-components/src/accordion-item/index.ts [hunk 5]
```diff
@@ -6,13 +6,13 @@ import {
 import { accordionItemStyles as styles } from "./accordion-item.styles";
 
 /**
- * The FAST Accordion Item Element. Implements {@link @microsoft/fast-foundation#AccordionItem},
- * {@link @microsoft/fast-foundation#accordionItemTemplate}
+ * A function that returns a {@link @microsoft/fast-foundation#AccordionItem} registration for configuring the component with a DesignSystem.
```
Comment (janechu) [inline packages/web-components/fast-components/src/accordion-item/index.ts:9]: Is there another link we can add for the `DesignSystem`?

**Review by janechu — APPROVED** [review]

**Review by nicholasrice — APPROVED** [review]

File: packages/web-components/fast-components/src/accordion-item/index.ts (hunk 5 above)
Comment (chrisdholt) [reply packages/web-components/fast-components/src/accordion-item/index.ts:9]: Good question - @nicholasrice or @EisenbergEffect, would `{@link @microsoft/fast-foundation#DesignSystem}` be accurate here? I can update all.

**Review by EisenbergEffect — APPROVED** [review]

File: packages/web-components/fast-components/src/accordion-item/index.ts (hunk 5 above)
Comment (nicholasrice) [reply packages/web-components/fast-components/src/accordion-item/index.ts:9]: yep, that's the link

**Pull request merged**